src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -1497,3 +1497,87 @@ will end up as internal server errors in the API.
 In general, do not ``return`` error responses in the REST API. They will be
 returned by the  error handler. Instead, raise the appropriate exception.
 
+Plug-ins
+~~~~~~~~
+
+New functionality can be provided by means of a plug-in architecture. Among the
+benefits this approach brings in, loosely coupled development is one of the most
+notable. As the Ceph-Dashboard grows in feature richness, its code-base becomes
+more and more complex. The hook-based nature of a plug-in architecture allows to
+extend functionality in a controlled manner, and isolate the scope of the
+changes.
+
+Ceph-dashboard relies on `Pluggy <https://pluggy.readthedocs.io>`_ to provide
```

Comment:
`Ceph-Dashboard` instead of `Ceph-dashboard` ?

Reply:
Thanks, @p-na! I'll fix it.